enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -334,31 +441,54 @@ spec:
     port: 443
 ```
 
-In this configuration, the Gateway specifies that it uses the
-"istio-ingressgateway" service and defines three listeners with two different
-host names: `*.stage.example.com` and `*.prod.example.com`.  Suppose the
-"istio-ingressgateway" has the following service (again, some extraneous details
-are abbreviated):
+In this configuration, the Gateway defines three listeners with two different
+host names: `*.stage.example.com` and `*.prod.example.com`, as well as two
+different TCP ports: 80 and 443.  For this Gateway, Istio creates the following
+service (again, some extraneous details are abbreviated in the following
+example):
 
 ```yaml
 apiVersion: v1
 kind: Service
 metadata:
-  name: istio-ingressgateway
+  labels:
+    gateway.istio.io/managed: openshift.io-gateway-controller
+  name: example-gateway
   namespace: openshift-ingress
+  ownerReferences:
+  - apiVersion: gateway.networking.k8s.io/v1alpha2
+    kind: Gateway
+    name: example-gateway
 spec:
   ports:
-  - name: http
+  - name: status-port
+    port: 15021
+  - name: stage-http
     port: 80
-  - name: https
+  - name: stage-https
```

Comment:
That's a little bit confusing example.  Maybe note that the ports are in use by two listeners, but got named for only one of the listeners.